scripts/scripts.js

```diff
@@ -639,7 +639,7 @@ export function buildArticleCard(article, type = 'article') {
 
   const path = article.path.split('.')[0];
 
-  const picture = createOptimizedPicture(image, imageAlt || title, false, [{ width: '750' }]);
+  const picture = createOptimizedPicture(image, imageAlt || title, type === 'featured-article', [{ width: '750' }]);
```

Comment:
Why only the `featured-article` and not the `article-header` too ?
We could change the `buildArticleCard` method signature and an extra optional boolean parameter `loadImage`. Like this, the block can decide if it want the image immediately or not. Which tends to say that there will be blocks that know they are LCP blocks...